gradle/versions.gradle

```diff
@@ -6,7 +6,8 @@ ext {
     compile_sdk_version = 28
     build_tools_version = '28.0.3'
 
-    support_lib = '27.1.1'
+    android_x_annotation = '1.1.0-alpha01'
```

Comment:
Alpha versions? Think this should be risky to use them, are there 1.0.x releases?

Reply:
yep, changed to stable